app/assets/javascripts/components/groups/groups_card.es6.jsx

```diff
@@ -5,14 +5,62 @@ class GroupsCard extends Component {
   // --------------------------------------------------
   static get propTypes() {
     return {
+      editable: React.PropTypes.bool.isRequired,
       group: React.PropTypes.object.isRequired,
       media: React.PropTypes.string.isRequired,
     };
   }
 
+
+  // --------------------------------------------------
+  // Styles
+  // --------------------------------------------------
+  get styles() {
+    return {
+      header: {
+        display: 'flex',
+        justifyContent: 'flex-end',
+        width: '100%',
+        marginBottom: '5px',
+        borderBottom: 'solid #D6D6D6 1px',
+      }
+    }
```

Comment:
Don't forget your commas and semicolons @sonia-y, I'll fix it for you this time!